### DISCUSSION TITLE
Use MNE study template code

### DISCUSSION THREAD

**Discussion by JohnGriffiths** [NeuroTechX/EEG-ExPy#73] [Ideas]
We should try give the MNE study template a spin on our datasets

https://github.com/mne-tools/mne-study-template

We should also consider using some of the preprocessing steps by default in all analyses. 

Anyone wanting to play around with analyses - take a look and let us know your thoughts!

**Comment by ErikBjare** [comment]
I've looked into this before, but had some issues running it on the BIDS I produced from my own data. (I tracked it here: https://github.com/ErikBjare/thesis/issues/10)

Didn't give it a second shot, but if someone wants to look into this I tried to do the BIDS conversion here: https://github.com/ErikBjare/thesis/blob/master/src/eegwatch/bids.py (the output of which seems to pass the https://github.com/bids-standard/bids-validator)
